app/lib/actions/katello/content_view/publish.rb

```diff
@@ -5,10 +5,12 @@ module ContentView
       class Publish < Actions::EntryAction
         include ::Katello::ContentViewHelper
         attr_accessor :version
-        # rubocop:disable Metrics/MethodLength,Metrics/AbcSize,Metrics/CyclomaticComplexity
+        # rubocop:disable Metrics/MethodLength,Metrics/AbcSize,Metrics/CyclomaticComplexity,Metrics/PerceivedComplexity
```

Comment:
Rubocop has a good point here.  Is it possible / would it make sense to refactor the import-specific logic into a separate action?

Reply:
This would probably also make it easier to test

Reply:
I don't want to duplicate the logic of publish verbatim just because 3 lines are different between import and publish. And more over Import is a separate action that calls publish setting the import_only flag.